en/docs/reference/connectors/cdc-inbound-endpoint/cdc-inbound-endpoint-example.md

```diff
@@ -0,0 +1,154 @@
+
+## What you'll build
+
+This scenario demonstrates how the CDC inbound endpoint can be used to capture changes occurring in a MySQL database. In this example, we will configure the **CDC Inbound Endpoint** to capture changes made to a **MySQL** table, such as insertions, updates, and deletions.
+
+In this example, a relational database table is used to store product information. The product data is added to the database by an external system that is outside of the enterprise's control. As soon as a new product is inserted, the system needs to detect and process the data. The integration runtime is used here to listen for database changes and trigger the relevant processes. It can either invoke backend APIs or place the data onto a message bus after performing the necessary data transformations. However, for the sake of simplicity in this example, we will simply log the message. You can extend this example as needed using WSO2 mediators.
+
+## Set up the environment 
+
+1. First, install [MySQL database](https://www.mysql.com/) locally. If you have a remote server, please obtain the credentials required to connect.  
+
+2. Connect to MySQL server.
+
+3. Create a database called `inventory`. 
+    ```sql
+    CREATE DATABASE inventory;
+    ```
+
+4. Then create a table called `products` under that database using the following SQL script. 
+    ```sql
+    CREATE TABLE products (
+        id INT AUTO_INCREMENT PRIMARY KEY,
+        name VARCHAR(255) NOT NULL,
+        price REAL NOT NULL
+    );
+    ```
+
+6. Download the latest Debezium Orbit JAR for **2.1.4.Final.wso2v2** from [nexus](https://maven.wso2.org/nexus/content/repositories/public/org/wso2/orbit/debezium/debezium/) and place it in `<Project Home>/deployment/libs` directory.
+
+7. Download the JDBC driver from the [MySQL website](https://dev.mysql.com/downloads/connector/j/), and copy it into the `<Project Home>/deployment/libs` directory.
+
+8. Create a user. For detailed instructions, refer to the [MySQL Creating User](https://debezium.io/documentation/reference/stable/connectors/mysql.html#mysql-creating-user) documentation. 
+
+8. Enable binlog. For detailed instructions, refer to the [Enable MySQL Binlog](https://debezium.io/documentation/reference/stable/connectors/mysql.html#enable-mysql-binlog) documentation. 
+
+9. Enable GTIDs. For detailed instructions, refer to the [Enable MySQL GTIDs](https://debezium.io/documentation/reference/stable/connectors/mysql.html#enable-mysql-gtids) documentation. 
+
+!!! info 
+    If you try a different database, the respective JDBC driver has to be downloaded and placed into the `<Project Home>/deployment/libs` directory. 
+
+## Configure inbound endpoint using micro integrator
+
+{!includes/create-new-project.md!}
+
+4. Create a sequence to process the capturing of a data change. 
```

Comment:
Fix the numbering

Reply:
The file `includes/create-new-project.md` provides the first 3 steps; therefore, I started from step 4 onward.